recipe/meta.yaml

```diff
@@ -15,6 +15,7 @@ source:
 
 build:
   number: 0
+  noarch: generic
```

Comment:
This won't work AFAIK unless we change the activation scripts in OpenSSL.

Reply:
Or more, it will work, but it will break stuff that rely on ca-certificates.

Reply:
Oh that was unintended leftover from past local changes. Thanks for the catch.